src/app/core/services/status.service.ts

```
@@ -3,23 +3,34 @@
+    private apiService: ApiService,
+    private orgUserSettingsService: OrgUserSettingsService,
+    private timezoneService: TimezoneService
+  ) {}
 
   find(objectType: string, objectId: string): Observable<ExtendedStatus[]> {
-    return this.apiService.get('/' + objectType + '/' + objectId + '/estatuses').pipe(
-      map((estatuses: ExtendedStatus[]) =>
-        estatuses?.map((estatus) => {
-          estatus.st_created_at = new Date(estatus.st_created_at);
+    return forkJoin([
+      this.apiService.get<ExtendedStatus[]>('/' + objectType + '/' + objectId + '/estatuses'),
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)